### PR TITLE
Binary Matrix 

### PR DESCRIPTION
**Pull request by soudabeh19**
(Option -b) Binary Matrix to represent the existence of difference in conditions pair according to the subject and file

### CODE REVIEW

File: verifyFiles.py
Comment (glatard) [inline verifyFiles.py]: `flag` is not a good name for a parameter -> `return_detailed_differences`.

File: verifyFiles.py
Comment (glatard) [inline verifyFiles.py]: A short comment would be appreciated.

File: verifyFiles.py
Comment (glatard) [reply verifyFiles.py]: You should also document this in the function's comments.

File: verifyFiles.py
Comment (glatard) [inline verifyFiles.py]: I don't think you need to loop twice over `path_names`.

File: verifyFiles.py
Comment (glatard) [inline verifyFiles.py]: how about returning both anyways and removing the flag?

File: verifyFiles.py
Comment (glatard) [inline verifyFiles.py]: there's a new typo here.

File: verifyFiles.py
Comment (glatard) [inline verifyFiles.py]: new typo here too.

File: verifyFiles.py
Comment (glatard) [inline verifyFiles.py]: don't capitalize variables (`Diff`).

**Review by glatard — CHANGES_REQUESTED** [review]
Nice work! Only minor changes are required, see inside comments.

File: verifyFiles.py
Comment (soudabeh19) [reply verifyFiles.py]: sure

File: verifyFiles.py
Comment (soudabeh19) [reply verifyFiles.py]: you are right

File: verifyFiles.py
Comment (soudabeh19) [reply verifyFiles.py]: sure, I'll do

File: verifyFiles.py
Comment (soudabeh19) [reply verifyFiles.py]: nice I'll do

File: verifyFiles.py
Comment (soudabeh19) [reply verifyFiles.py]: Thanks for your detailed review

File: verifyFiles.py
Comment (glatard) [inline verifyFiles.py]: You don't need the last parameter now that you always return both matrices.

**Review by glatard — CHANGES_REQUESTED** [review]
Just a small remaining comment.

File: verifyFiles.py
Comment (soudabeh19) [reply verifyFiles.py]: Oops! totally right

**Comment by glatard** [comment]
Hi @lalet, could you review this PR and make sure your stuff still works with these modifications?

**Review by glatard — APPROVED** [review]
Good to go! I'll just wait for @lalet's feedback before merging.

File: verifyFiles.py
Comment (lalet) [inline verifyFiles.py]: Could you please correct the typos in the help section of binaryMatix

**Review by lalet — CHANGES_REQUESTED** [review]
Good work @soudabeh19 !! 
I have requested for a few minor changes due to typos. 
If you have added a new parameter, I would recommend you to add it to the readme page as well.

File: verifyFiles.py
Comment (soudabeh19) [reply verifyFiles.py]: Thanks for mentioning

**Review by lalet — APPROVED** [review]

**Pull request merged**
